tests/test_faster_rcnn.py

```diff
@@ -47,6 +49,12 @@ def config():
 def test_faster_rcnn_train_one_epoch(config, dataset):
     """test train one epoch."""
     writer = MagicMock()
+
+    # XXX This is just a hot fix to prevent a mysterious folder such as:
+    # <MagicMock name='mock.logdir' id='140420520377936'> showed up after
```

Comment:
Whats the purpose of adding instructions here? Cant we create a task and fix it?

Reply:
This fixes issue when writer is mocked but the FasterRCNN constructor tries to use the `log_dir` stored in the mocked writer. I will create a Github issue.